FacebookWinFormsApp/FormMain.cs

```diff
@@ -45,15 +45,17 @@ private void loadUserInfo()
             listBoxAboutInfo.Items.Add(m_LoggedInUser.Birthday);
             listBoxAboutInfo.Items.Add(userGender);
             listBoxAboutInfo.Items.Add(r_Astrology.GetZodiac(m_LoggedInUser.Birthday));
+            labelUserName.Text = m_LoggedInUser.Name;
+            labelUserName.Show();
 
             fetchCoverPhoto();
             fetchAlbums();
-            fetchAlbumPhotos();
+            //fetchAlbumPhotos();
             fetchUserPosts();
-            fetchFriendsList();
+            //fetchFriendsList();
             fetchPagesList();
             fetchGroupsList();
-            fetchEventsList();
+           // fetchEventsList();
```

Comment:
In comment only couse the App run faster, and forgot to take out of comment before push